src/LibFLExBridge-ChorusPlugin/Infrastructure/FlexBridgeConstants.cs

```diff
@@ -72,9 +72,36 @@ internal static class FlexBridgeConstants
 		internal const string RtTag = "rt";
 
 		// Model Version
+		// An extension for a file that stores {"modelversion":<unit that is the FW model version>}
 		internal const string ModelVersion = "ModelVersion";
+		// The full name of that file
 		internal const string ModelVersionFilename = "FLExProject." + ModelVersion;
 
+		// A value that specifies a version of FlexBridge that affects the data written to the
+		// repo, but not the FieldWorks model version. This is part of the branch name, so for
+		// example a version of FlexBridge that is looking for data on branch 700072 or 700072.1 will not
+		// try to read data on branch 700072.2, even though (if it could PutHumptyTogetherAgain)
+		// its caller would be able to handle 700072 model data. This allows us to change how
+		// FlexBridge handles the data without changing the data model version (which affects other
+		// products and is becoming increasingly expensive to change).
+		// Note that send/receive never tries to handle data on other branches; it only merges with
+		// data on its own branch (though warning if there are new commits on other branches).
+		// Data migration only happens when a new version of FLEx (and FlexBridge) is installed
+		// on a computer that has the data needing migration, and upgrades and Sends it;
+		// or when making a new clone of the project using a newer version of FLEx.
+		// Because of this clone-and-upgrade possibility, FlexBridge must be able to
+		// PutHumptyTogetherAgain for both old data models and old Humpty strategies.
```

Comment:
> FlexBridge must be able to PutHumptyTogetherAgain for both old data models and old Humpty strategies.

This contradicts the implementation in `src/FLEx-ChorusPlugin/Infrastructure/ActionHandlers/ObtainProjectStrategyFlex.cs` where we only deal with the current `FlexBridgeDataVersion` (assuming that the Humpty strategy corresponds to the FlexBridgeDataVersion).